__tests__/getEndpoints.test.js

```diff
@@ -36,4 +36,52 @@ describe("GET endpoints", () => {
         });
     });
   });
+  describe("GET /api/articles/:article_id", () => {
+    test("200: responds with one article object with correct keys", () => {
+      const desiredKeyArray = [
+        "title",
+        "topic",
+        "author",
+        "body",
+        "created_at",
+        "article_img_url",
+      ];
+      return request(app)
+        .get("/api/articles/2")
+        .expect(200)
+        .then((res) => {
+          expect(typeof res.body.article).toBe("object");
+          expect(Array.isArray(res.body.article)).toBe(false);
```

Comment:
I like this test with its use of .every to check property names.  However, you can also check strict equality to an object (in this case you know what article with id 2 is going to look like, so can just test that you are getting back the expected object, with all correct keys and values.